src/components/Header/DropDown.js

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect, useState } from "react";
+import python from "./asset/python.svg";
+import typing from "./asset/typing.svg";
+import web from "./asset/web.svg";
+import language from "./asset/language.svg";
+import residential from "./asset/residential.svg";
+import random from "./asset/random.svg";
+import { Link } from "react-router-dom";
+import { PATHS, interpolatePath } from "../../constant";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import useStyles from "./styles";
+import { useSelector, useDispatch } from "react-redux";
+import { actions as pathwayActions } from "../PathwayCourse/redux/action";
+
+import {
+  Typography,
+  Menu,
+  MenuItem,
+  CardContent,
+  Divider,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+} from "@mui/material";
+
+const students = {
+  image: [python, typing, web, language, residential, random],
+  Learn: [
+    { title: "Python", code: "PRGPYT", type: "internal" },
+    { title: "Typing Guru", code: "TYPGRU", type: "internal" },
+    { title: "JavaScript", code: "JSRPIT", type: "internal" },
+    { title: "English", code: "SPKENG", type: "internal" },
+    {
+      title: "Residential Programmes",
+      path: PATHS.RESIDENTIAL_COURSE,
+      type: "internal",
+    },
+    {
+      title: "Open Courses",
+      path: PATHS.MISCELLANEOUS_COURSE,
+      type: "internal",
+    },
+  ],
+  About: [
+    { title: "Meraki Team", path: PATHS.MERAKI_TEAM, type: "internal" },
+    { title: "Our Story", path: PATHS.OUR_STORY, type: "internal" },
+  ],
+  GetInvolved: [
+    { title: "Become a Partner", path: PATHS.OUR_PARTNER, type: "internal" },
+    {
+      title: "Donate",
+      path: "https://www.navgurukul.org/donate",
+      type: "external",
+    },
+    {
+      title: "Careers",
+      path: "https://recruiterflow.com/navgurukul/jobs",
+      type: "external",
+    },
+  ],
+};
+
+export const MobileDropDown = ({ Menu, handleClose, toggleDrawer }) => {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const { data } = useSelector((state) => state.Pathways);
+
+  useEffect(() => {
+    dispatch(pathwayActions.getPathways());
+  }, [dispatch]);
+
+  data &&
+    data.pathways.forEach((pathway) => {
+      students.Learn.forEach((item) => {
+        if (pathway.code === item.code) {
+          item["id"] = pathway.id;
```

Comment:
Generally, you use dot notation when applicable (e.g., when the property names are valid variable names) 
```suggestion
          item.id = pathway.id;
```